### DISCUSSION TITLE
openelb- keepalived-vip configmap openelb-vip-configmap was not found

### DISCUSSION THREAD

**Discussion by rkazak07** [openelb/openelb#353] [General]
Hello, I'm getting the following error in the keepalived-vip pods on openelb. if I delete "nfs-provisioner ", errors are starting to occur again for different pods. How do I need to move forward with this issue.

k8s version: v1.24.6
kube-keepalived-vip:0.35
openelb:v0.5.0

LOGS:
W0810 11:55:44.047666       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:46.047519       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:48.047576       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:50.047560       7 main.go:90] requeuing cattle-fleet-system/gitjob, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:52.047537       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:54.047469       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:56.047563       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:55:58.047549       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:00.047475       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:02.047567       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:04.047586       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:06.047671       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:08.047702       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:10.047590       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:12.047688       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:14.047606       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:16.047552       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:18.047503       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:20.047729       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:22.047583       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:24.047723       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:26.047712       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:28.047592       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:30.047615       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:32.047529       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:34.047559       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:36.047637       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:38.047715       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:40.047538       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:42.047571       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:44.047488       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:46.047735       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:48.047728       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:50.047656       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
W0810 11:56:52.047713       7 main.go:90] requeuing nfs-provisioner/cluster.local-nfs-subdir-external-provisioner, err unexpected error searching configmap openelb-izmir/openelb-vip-configmap: configmap openelb-vip-configmap was not found
^C

**Comment by renyunkang** [comment]
It doesn't matter if you don't use vip mode. You can also manually create an empty `configmap` `openelb-izmir/openelb-vip-configmap` to make it stop reporting errors.

**Comment by rkazak07** [comment]
Hello, I have installed kafka for openelb, I created the applications when loadbalancer clicked. But when I delete the apps, their threads still seem to be active, there are no kafka apps in namespace right now, and how can I delete them. Thanks
![iplist](https://github.com/openelb/openelb/assets/106657209/a1dd054e-65ee-4528-8036-b86c15331a40)
